src/components/Table/_table.scss

```diff
@@ -1 +1,23 @@
 @import 'TableHead/table-head';
+@import '~carbon-components/scss/components/data-table/data-table';
+
+.toast {
+  margin-bottom: 0.5rem;
+  min-width: 30rem;
+  position: fixed;
+  top: 0;
+  right: 0;
+  z-index: 1;
+}
+
+.buttons {
```

Comment:
Class names such as `toast` and `buttons` are too generic and could clash in the global namespace. We should be using BEM syntax for our classnames or at the very least qualified as a button of the tableheader or wherever these classes are used.